bulldozer/signals.go

```diff
@@ -105,13 +105,15 @@ func (s *Signals) MatchesAll(ctx context.Context, pullCtx pull.Context, tag stri
 	}
 
 	for _, signal := range signals {
-		matches, _, err := signal.Matches(ctx, pullCtx, tag)
-		if err != nil {
-			return false, "", err
-		}
+		if signal.Enabled() {
```

Comment:
Just checking: if no triggers/signals are specified in the configuration (i.e. `Enabled()` always returns false), `MatchesAll` will return true. Is that the intended behavior? I can see valid arguments for both `true` and `false` here.

Reply:
I think documenting the behaviour might be nice either way. I think similar to #217, we may want to either provide an option to opt out of the behaviour. I think the current behaviour is good for now though.

Reply:
If there are no triggers/signals, then yes MatchesAll will be true. I agree this could go either way, although if there are no signals/triggers defined that is an invalid config as it is currently defined. I'll go ahead and make that return `false` and update the docs.